### PR TITLE
Refactor client side js into folders for apps and files for components

### PR DESCRIPTION
**Pull request by chrisuehlinger**
To address issue #2 I've created a folder for each app (camera, projector and editor) and I've broken each React component into its own file. I tried to use `git mv` as much as possible so the moved files would be connected to the original files' git histories. I also went through and updated the references to things like utils and constants so they'd be where they were needed and tested the result to make sure it compiled and worked.

Please let me know if you have any critiques or concerns, and I'd be happy to address them!

### CODE REVIEW

File: client/camera/CameraVideo.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:local(.keyPoint) {
+    position: absolute;
+    background: rgba(255, 255, 255, 0.3);
+    border-radius: 1000px;
+    cursor: pointer;
+  }
+  :local(.keyPoint):hover {
+    background: rgba(255, 255, 255, 0.6);
+  }
```
Comment (janpaul123) [inline client/camera/CameraVideo.css]: Indentation seems off?

File: client/camera/Knob.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    margin-top: calc(size / -2);
+    width: size;
+    height: size;
+    box-shadow: 0 0 0 1px red;
+    border-radius: 1000px;
+    cursor: pointer;
+  }
+  :local(.knob):hover {
+    background: rgba(255, 0, 0, 0.5);
+  }
```
Comment (janpaul123) [inline client/camera/Knob.css]: Here too

File: client/camera/CameraVideo.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+import React from 'react';
+
+import detectPrograms from './detectPrograms';
+import styles from './CameraVideo.css';
+
+import Knob from './Knob';
+
+export default class CameraVideo extends React.Component {
+    constructor(props) {
```
Comment (janpaul123) [inline client/camera/CameraVideo.js]: Weird indentation here too? What's going on 😛

**Comment by chrisuehlinger** [comment]
My editor must’ve been acting up and autoformatting things, I’ll sort this out.

**Comment by janpaul123** [comment]
Haha thanks! 😄

**Comment by janpaul123** [comment]
I fixed this up myself and merged this PR manually. Thanks! 👍

**Pull request closed without merging**